protocol-jackson/src/main/java/io/camunda/zeebe/protocol/jackson/record/ZeebeStyle.java

```diff
@@ -28,7 +28,7 @@
     jdkOnly = true,
     defaults = @Value.Immutable(lazyhash = true, copy = false),
     validationMethod = ValidationMethod.NONE,
-    defaultAsDefault = true,
+    defaultAsDefault = false,
```

Comment:
`defaultAsDefault` is false by default 😄 

🔧 To avoid confusion, just remove it altogether.

Reply:
Already done ;)